docs/notebooks/sklearn_wrapper.ipynb

```diff
@@ -233,72 +239,163 @@
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "GridSearchCV(cv=5, error_score='raise',\n",
+       "       estimator=SklearnWrapperLdaModel(alpha='symmetric', chunksize=2000, corpus=None,\n",
+       "            decay=0.5, eta=None, eval_every=10, gamma_threshold=0.001,\n",
+       "            id2word=<gensim.corpora.dictionary.Dictionary object at 0x7fb82cfbb7d0>,\n",
+       "            iterations=50, minimum_probability=0.01, num_topics=5,\n",
+       "            offset=1.0, passes=20, random_state=None, update_every=1),\n",
+       "       fit_params={}, iid=True, n_jobs=1,\n",
+       "       param_grid={'num_topics': (2, 3, 5, 10), 'iterations': (1, 20, 50)},\n",
+       "       pre_dispatch='2*n_jobs', refit=True, return_train_score=True,\n",
+       "       scoring=<function scorer at 0x7fb82cfaf938>, verbose=0)"
+      ]
+     },
+     "execution_count": 32,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "obj=SklearnWrapperLdaModel(id2word=dictionary,num_topics=5,passes=20)\n",
+    "parameters = {'num_topics':(2, 3, 5, 10), 'iterations':(1,20,50)}\n",
+    "model = GridSearchCV(obj, parameters, scoring=scorer, cv=5)\n",
+    "model.fit(corpus)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'iterations': 50, 'num_topics': 3}"
+      ]
+     },
+     "execution_count": 33,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "model.best_params_"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Example of Using Pipeline"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": 34,
    "metadata": {
     "collapsed": true
    },
    "outputs": [],
    "source": [
-    "def print_features(clf, vocab, n=10):\n",
+    "from sklearn.pipeline import Pipeline\n",
+    "from sklearn import linear_model\n",
+    "def print_features_pipe(clf, vocab, n=10):\n",
     "    ''' Better printing for sorted list '''\n",
-    "    coef = clf.coef_[0]\n",
+    "    coef = clf.named_steps['classifier'].coef_[0]\n",
+    "    print coef\n",
     "    print 'Positive features: %s' % (' '.join(['%s:%.2f' % (vocab[j], coef[j]) for j in np.argsort(coef)[::-1][:n] if coef[j] > 0]))\n",
-    "    print 'Negative features: %s' % (' '.join(['%s:%.2f' % (vocab[j], coef[j]) for j in np.argsort(coef)[:n] if coef[j] < 0]))"
+    "    print 'Negative features: %s' % (' '.join(['%s:%.2f' % (vocab[j], coef[j]) for j in np.argsort(coef)[:n] if coef[j] < 0]))\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 35,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "id2word=Dictionary(map(lambda x : x.split(),data.data))\n",
```

Comment:
`map` is discouraged -- use comprehensions and generators.

Also, PEP8 -- space after comma, spaces around ` = `.